### PR TITLE
render 5 recipes

### PR DESCRIPTION
**Pull request by ihorhladkov**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
   // const counter = useCountStore((state) => state.counter);
   // const increase = useCountStore((state) => state.increase);
   // const form = useForm();
+  // const { data } = api.categories.greeting.useQuery();
+
+  // const { data: recipes } = api.categoriesRouter.getCategory.useQuery();
+  const { data: recipeData, isLoading} = api.recipesRouter.getSortedRecipes.useQuery();
+
+  if (recipeData) {
+    console.log(recipeData);
```
Comment (Nordexz) [inline src/app/page.tsx:18]: remove

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
-      {/* <Button variant="ghost">Button</Button> */}
       {/* <h1>Books: {counter} </h1> */}
-      <div className="grid grid-cols-4 gap-5">
-        <Card />
-        <Card />
-        <Card />
-        <Card />
-        <Card />
+      <div className="grid grid-cols-2 gap-5">
+        {!isLoading ?
```
Comment (Nordexz) [inline src/app/page.tsx:25]: try to use useSuspenseQuery and add loading.tsx file

**Review by Nordexz — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 2 above)
Comment (ihorhladkov) [reply src/app/page.tsx]: https://recipes-il7ijkxaq-ihors-projects-85c75ba3.vercel.app/

**Comment by Nordexz** [comment]
adaptive + scrollable ingredients block

**Pull request merged**